apollo-federation/src/operation/mod.rs

```diff
@@ -785,6 +784,10 @@ impl Selection {
         }
     }
 
+    fn sub_selection_type_position(&self) -> Option<CompositeTypeDefinitionPosition> {
```

Comment:
Just cleanup: this function exists on, and was used as  `.element().sub_selection_type_position()`, but doing that requires creating an `OperationElement` (in JS the main structures are built on top of `OperationElement`, but in Rust they are not, so it requires conversion Arc copies). The exact same data already exists on the selection set type so we can just return it directly without that conversion.